### ISSUE TITLE
Creating TypeScript transform, transpiled imports fail to load

### ISSUE THREAD

**Issue by EFXAlexLong** [systemjs/systemjs#1974]
Hello,

I'm not sure how to use the plugin-typescript with SystemJS 2.0.0, and I'm not sure that it's even compatible. I began to work on a TS transform, based on babel-transform, to learn SystemJS and to be able to learn to code Angular 8 apps w/o the usual CLI build tools, meaning I would transpile the project in the browser.

I think I'm getting warmer, but now I'm trying to resolve an issue with imports (and probably exports, also) of Angular modules, such as "@angular/core", "@angular/platform-browser-dynamic", etc. My TS files transpile, register, and execute successfully. Meanwhile, the Angular *.umd.js files are fetched and transformed to SystemJS format, but the dependencies won't execute. 

I've created a Plunker of what I've got so far, and any advice would be appreciated.

[Plunker](https://next.plnkr.co/edit/IjbxOl1BR3TXpWBw)

**Comment by EFXAlexLong** [comment]
Well, I got my code to work! Kind of.

First, I added a conditional to my transformer to try to ignore UMD and AMD formatted files. I wasn't sure if there was a method within TypeScript that I could use to determine the file format properly, but that's something I could ask on StackOverflow or keep digging into it. However, ignoring UMD/AMD only worked so far, until I ran into an issue with the URL for an external dependency (ex, "@angular/core") being changed to `http://localhost:3000/@angular/<dep>`. This caused a 404 error when being fetched in the transform extra.

Workaround (?): Instead of loading Angular packages from the `https://unpkg.com/@angular/<dep>@<ver>/bundles` folder, I tried configuring the import-maps to point to the `esm2015` folder. Loading the test app was successful but very slow from transpiling ALL THE THINGS. Not the solution I'm looking for, so I will keep troubleshooting. Again, I appreciate if anyone has the time to put fresh eyes on the Plunker above and provide any feedback.
